hack/lib/util.sh

```diff
@@ -136,14 +136,21 @@ kube::util::gen-doc() {
   # create the list of generated files
   ls "${tmpdir}" | LC_ALL=C sort > "${tmpdir}/.files_generated"
 
-  # remove all old generated file from the destination
   while read file; do
+    # Add analytics link to generated .md files
+    if [[ "${file}" == *".md"* ]]; then
+      local link destroot
+      destroot=$(dirname ${dest})
+      link=$( kube::util::analytics-link "${dest#$destroot/}${file}" )
+      printf "\n${link}\n" >> "${tmpdir}/${file}"
```

Comment:
not a big fan of `printf`... What if `${link}` has a `%` in it? I'd recommend using `echo` instead, maybe `echo -en` which will not add a newline and interpret your `\n`s and behave approximately like your printf does, though if you're adding a newline to the end anyways you could skip the `-n` too.